website/docs/r/network_connection_monitor.html.markdown

```diff
@@ -74,41 +157,131 @@ The following arguments are supported:
-
 * `location` - (Required) The Azure Region where the Network Connection Monitor should exist. Changing this forces a new Network Connection Monitor to be created.
 
-* `destination` - (Required) A `destination` block as defined below.
+* `network_watcher_id` - (Required) The ID of the Network Watcher. Changing this forces a new Network Connection Monitor to be created.
 
-* `network_watcher_name` - (Required) The name of the Network Watcher. Changing this forces a new Network Connection Monitor to be created.
+* `endpoint` - (Required) A `endpoint` block as defined below.
 
-* `source` - (Required) A `source` block as defined below.
+* `test_configuration` - (Required) A `test_configuration` block as defined below.
+
+* `test_group` - (Required) A `test_group` block as defined below.
 
 ---
 
-* `auto_start` - (Optional) Will the connection monitor start automatically once created? Changing this forces a new Network Connection Monitor to be created.
+* `notes` - (Optional) Any notes about the Network Connection Monitor.
 
-* `interval_in_seconds` - (Optional) Monitoring interval in seconds.
+* `output_workspace_resource_ids` - (Optional) A list of the log analytics workspace id that should be used for producing output into a log analytics.
 
 * `tags` - (Optional) A mapping of tags which should be assigned to the Network Connection Monitor.
 
 ---
 
-A `destination` block supports the following:
+A `endpoint` block supports the following:
+
+* `name` - (Required) The name of the connection monitor endpoint.
+
+* `address` - (Optional) The address of the connection monitor endpoint (IP or domain name).
+
+* `filter` - (Optional) A `filter` block as defined below.
+
+* `virtual_machine_id` - (Optional) The ID of the virtual machine which is used as the endpoint by connection monitor.
+
+---
+
+A `filter` block supports the following:
+
+* `type` - (Optional) The behavior type of this endpoint filter. Currently the only allowed value is `Include`. Defaults to `Include`.
+
+* `item` - (Optional) A `item` block as defined below.
 
-* `port` - (Required) The destination port used by connection monitor.
+---
+
+A `item` block supports the following:
 
-* `address` - (Optional) The address of the connection monitor destination (IP or domain name). Conflicts with `destination.0.virtual_machine_id`
+* `type` - (Optional) The type of items included in the filter. Possible values are `AgentAddress`. Defaults to `AgentAddress`.
 
-* `virtual_machine_id` - (Optional) The ID of the virtual machine used as the destination by connection monitor. Conflicts with `destination.0.address`
+* `address` - (Optional) The address of the filter item.
 
 ---
 
-A `source` block supports the following:
+A `test_configuration` block supports the following:
+
+* `name` - (Required) The name of the connection monitor test configuration.
```

Comment:
Can we capitalize the first letter of the resource type to be consistent?

Reply:
updated